apps/watch/src/components/Footer/Footer.tsx

```diff
@@ -117,14 +121,16 @@ export function Footer({ isHome = false }: FooterProps): ReactElement {
           }}
         >
           <Stack direction="row" spacing="100px">
-            <Typography variant="body2">Copyright © 1995-2022</Typography>
-            <Typography variant="body2">Jesus Film Project®</Typography>
-            <Typography variant="body2">All rights reserved</Typography>
+            <Typography variant="body2">
+              {t('Copyright © 1995-2022')}
```

Comment:
```suggestion
              {t('Copyright © 1995-{{year}}', { year: new Date().getFullYear() } )}
```

This will mean we don't have to translate this every year.